src/snowflake/cli/app/dev/docs/project_definition_docs_generator.py

```diff
@@ -39,7 +39,7 @@ def generate_project_definition_docs(root: SecurePath):
 
     root.mkdir(exist_ok=True)
     list_of_sections = model_json_schema(
-        ProjectDefinition, schema_generator=ProjectDefinitionGenerateJsonSchema
+        _DefinitionV11, schema_generator=ProjectDefinitionGenerateJsonSchema
```

Comment:
It's a code smell to me to import something that's `_`-prefixed as in Python this indicates the item is internal to the file. WDYT?

Reply:
I agree, but the code here specifically tests for v1.1. It's just that before my change it was done explicitly.
I'll let the CLI team to review this area.

Reply:
I agree with @sfc-gh-cgorrie

Reply:
Can we change def `_get_version_map()` to be externally accesible? Renaming it as `get_version_map()`. we could use that function to generate the docs for multiple versions and to avoid explicitly accessing to file clases

Reply:
If we want to keep only generating docs for 1.1, I don't see the harm in making the definition classes public

Reply:
Removed the `_` prefix from the definition classes and the `get_version_map()` function